### PR TITLE
bug fix webhook response not sending data to tinybird

### PR DESCRIPTION
**Pull request by anamarn**
Solves https://github.com/twentyhq/private-issues/issues/118

**TLDR**

Fix webhook response not sending data to tinybird when the url is not a link.

**Changes in Tinybird:**

- Add column Success to webhook payload (boolean)
-  Changed the parameter WebhookIdRequest to WebhookId in the getWebhooksResponse api point.
-  Those changes can be seen in the tinybird workspace twenty_analytics_playground

**In order to test**

1. Set ANALYTICS_ENABLED to true
2. Set TINYBIRD_INGEST_TOKEN to your token from the workspace twenty_analytics_playground
3. Set TINYBIRD_GENERATE_JWT_TOKEN to the admin kwt token from the workspace twenty_analytics_playground
4. Set TINYBIRD_WORKSPACE_UUID to the UUID of twenty_analytics_playground
5. Create a Webhook in twenty and set wich events it needs to track
6. Run twenty-worker in order to make the webhooks work.
7. Do your tasks in order to populate the data
8.  Look at your webhooks in settings>api and webhooks> your webhook and the statistics should be displayed

### CODE REVIEW

File: packages/twenty-front/src/modules/settings/developers/webhook/utils/fetchGraphDataOrThrow.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const fetchGraphDataOrThrow = async ({
 }: fetchGraphDataOrThrowProps) => {
   const queryString = new URLSearchParams({
     ...WEBHOOK_GRAPH_API_OPTIONS_MAP[windowLength],
-    webhookIdRequest: webhookId,
+    webhookId: webhookId,
```
Comment (Weiko) [inline packages/twenty-front/src/modules/settings/developers/webhook/utils/fetchGraphDataOrThrow.ts:17]: redundant

File: packages/twenty-server/src/engine/api/graphql/workspace-query-runner/jobs/call-webhook.job.ts [hunk 2]
```diff
@@ -46,10 +47,11 @@ export class CallWebhookJob {
       const eventInput = {
```
Comment (Weiko) [inline packages/twenty-server/src/engine/api/graphql/workspace-query-runner/jobs/call-webhook.job.ts:47]: This part seems a bit redundant as well, shouldn't success be dependant on the response status / call failure? The rest seems very similar so I think it could be refactored a bit

**Review by Weiko — APPROVED** [review]
Left some nit comments but LGTM!

**Review by FelixMalfait — APPROVED** [review]
Thank you!
